types/json-schema/index.d.ts

```diff
@@ -16,7 +16,7 @@
  * @see https://tools.ietf.org/html/draft-zyp-json-schema-03#section-5.1
  */
 export type JSONSchema4TypeName =
-    | 'string'
+    | 'string' //
```

Comment:
What's with these empty comments?

Reply:
They're a marker for `prettier` to tell it to format these as list on their own line.